Optimization Online.js

```diff
@@ -0,0 +1,364 @@
+{
+	"translatorID": "0ee623f9-0a5c-4f6a-a5e1-10e2feaa16a7",
+	"label": "Optimization Online",
+	"creator": "Pascal Quach",
+	"target": "^https?://optimization-online\\.org/",
+	"minVersion": "6.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-06-04 09:53:10"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2023 Pascal Quach <pascal.quach@centralesupelec.fr>
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) { // eslint-disable-line no-unused-vars
+	if (doc.body.classList.contains('single-post')) {
+		return 'preprint';
+	}
+	else if (getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('.entry-title > a');
+	for (let row of rows) {
+		let href = row.href;
+		let title = ZU.trimInternal(row.textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+async function doWeb(doc, url) {
+	if (detectWeb(doc, url) == 'multiple') {
+		let items = await Zotero.selectItems(getSearchResults(doc, false));
+		if (!items) return;
+		for (let url of Object.keys(items)) {
+			await scrape(await requestDocument(url));
+		}
+	}
+	else {
+		await scrape(doc, url);
+	}
+}
+
+async function scrape(doc, url) {
+	let translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+
+	translator.setHandler('itemDone', (_obj, item) => {
+		// Item
+		item.itemType = 'preprint';
+		let title = doc.querySelector('h1.entry-title');
+		if (title) {
+			item.title = ZU.capitalizeTitle(title.innerText);
+		}
+		let tags = doc.querySelectorAll('div.entry-meta > span.tags-links > a[rel="tag"]');
+		if (tags.length) {
+			for (var i = 0; i < tags.length; i++) {
+				item.tags.push(ZU.cleanTags(tags[i].text));
+			}
+		}
+		let abstractNote = doc.querySelector('div.entry-content > div > p');
```

Comment:
ditto on using `text()`

Reply:
When I use `text()` instead, it duplicates mathjax in the abstract.
I left it as it is for now, would you know how to avoid that?
See https://optimization-online.org/?p=23021 for an example

Reply:
Thank you for the snippet @zoe-translates! I ended up using `innerText()` instead, which was simpler, thank you @AbeJellinek 
I also modified it for the title and added a test case.